crates/criterion_compat/criterion-0.5.1/Cargo.toml

```diff
@@ -152,7 +156,7 @@ default-features = false
 version = "0.8"
 
 [dev-dependencies.tempfile]
-version = "~3.5.0"
```

Comment:
We should specify the version in the workspace `Cargo.toml` and use {workspace = true} here, and change it for `crates/codspeed` aswell, like we did for `itertools`